neurokit2/ecg/ecg_quality.py

```diff
@@ -157,7 +157,7 @@ def _ecg_quality_averageQRS(ecg_cleaned, rpeaks=None, sampling_rate=1000):
 
     # Interpolate
     quality = signal_interpolate(
-        rpeaks, quality, x_new=np.arange(len(ecg_cleaned)), method="quadratic"
+        rpeaks, quality, x_new=np.arange(len(ecg_cleaned)), method="linear"
```

Comment:
```suggestion
        rpeaks, quality, x_new=np.arange(len(ecg_cleaned)), method="previous"
```

To make it consistent with https://github.com/neuropsychology/NeuroKit/pull/1044/files, maybe "previous" makes more sense

Reply:
Agreed. While "previous" doesn't yield as nice-looking plots as other interpolation methods, I think it better represents the design intent. All PPG quality methods and the ECG quality method in question (averageQRS) measures are calculated per heartbeat, so it makes sense to have the quality value reported per heartbeat. The interpolation to match the length of the raw ECG/PPG signal is a convenience step.